hazelcast/src/main/java/com/hazelcast/executor/impl/DistributedExecutorService.java

```diff
@@ -132,6 +155,14 @@ public boolean cancel(String uuid, boolean interrupt) {
         return false;
     }
 
+    public String getName(String uuid) {
+        CallableProcessor proc = submittedTasks.get(uuid);
+        if (proc != null) {
+            return proc.name;
+        }
+        return null;
```

Comment:
I believe this can cause a NPE if we try to cancel a non-existent UUID. This is because `com.hazelcast.spi.QuorumAwareService#getQuorumName` do not seem to allow a `null` parameter.

Reply:
Fixed.

Reply:
Can you just point me to where it was fixed? I can't seem to find it